examples/1.Ducks/PapaDuck/PapaDuck.ino

```diff
@@ -146,6 +155,9 @@ int quackJson(CdpPacket packet) {
   const int bufferSize = 4 * JSON_OBJECT_SIZE(4);
   StaticJsonDocument<bufferSize> doc;
 
+  std::vector<uint8_t> encrypted;
```

Comment:
You are not doing a check if encryption/decryption is enabled. This will break ducks not using encryption. There is a flag you can grab for if encryption is enabled. Might be better to take out all decryption stuff for now

Reply:
If we are going to go this route we need to add more functionality to the control pannel